main.py

```diff
@@ -1,14 +1,19 @@
+from bsoup import players_per_page_list, get_country_id
+from sql import DB
 import argparse
 from bs4 import BeautifulSoup
 import requests
-from bsoup import players_per_page_list, get_country_id
 import sys
 
 
 def filter_by_country(country, num):
+    db = DB('players.db')
```

Comment:
try to structure code such that the name of then functions, variables give correct hints about what the code does. 

Example here: 

function is called filter_by_country 
but what it does is scrapes html and writes things in a database 

A more semantically correct structure could be this
```
filters = parse_arguments
loop
  fetch_one_page 
  save_result_to_db 

```